pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
```

Comment:
oh I was really talking about on ``Styler``.

actually ``subset`` is quite useful and general (as you can simply give say a ``.select_dtype(...)`` as an indexer.

but I suspect what you really want is to provide formatters by dtypes a lot of the time (and alignments)